packages/components/src/segmented-control/test/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import { act, Simulate } from 'react-dom/test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { SegmentedControl } from '../index';
+
+describe( 'SegmentedControl', () => {
+	const options = (
+		<>
+			<SegmentedControl.Option value="olaf" label="O" />
+			<SegmentedControl.Option value="samantha" label="S" />
```

Comment:
Let's change these away from Frozen themed, just in case Disney decides they don't like it and sues WP 😬